### PR TITLE
Prosthetic repair standardization (#5823)

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Ports 5823

## Why It's Good For The Game

Just getting us up to date

## Changelog
```changelog
admin: Ports 5823 with modifications
tweak: adds one nanopaste to the medical wall vendors for 1000 cr
```

### CODE REVIEW

File: cev_eris.dme [hunk 1]
```diff
@@ -2820,6 +2820,7 @@
 #include "zzzz_modular_occulus\code\game\objects\items\plush.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\magnetic_lock.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\traitor_devices.dm"
+#include "zzzz_modular_occulus\code\game\objects\items\tools\_tools.dm"
```
Comment (Michiyamenotehifunana) [inline cev_eris.dme:2823]: Wait wha, what does this have to do with anything

Comment (Jamini) [reply cev_eris.dme:2823]: Welding limbs is in there.

Comment (Jamini) [reply cev_eris.dme]: I....don't even know where that came from. wtf is that?

**Review by Michiyamenotehifunana — APPROVED** [review]
Everything seems to check out, should be good.
